modules/openapi-generator/src/main/resources/aspnetcore/2.1/Startup.mustache

```diff
@@ -42,16 +42,16 @@ namespace {{packageName}}
         {
             // Add framework services.
             services
-                .AddMvc()
+                .AddMvc({{^useDefaultRoutng}}opts => opts.EnableEndpointRouting = false{{/useDefaultRoutng}})
                 {{#compatibilityVersion}}
                 .SetCompatibilityVersion(CompatibilityVersion.{{compatibilityVersion}})
                 {{/compatibilityVersion}}
-                .AddJsonOptions(opts =>
+                .{{#useNewtonsoft}}AddNewtonsoftJson{{/useNewtonsoft}}{{^useNewtonsoft}}AddJsonOptions{{/useNewtonsoft}}(opts =>
                 {
                     opts.SerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
                     opts.SerializerSettings.Converters.Add(new StringEnumConverter
                     {
-                        CamelCaseText = true
```

Comment:
this option is needed for older 2.2/2.1 framework version.

Reply:
It is generated for older frameworks - the code generated is conditional based on 2.1/2.2 or 3.0+
Eventually we might want to make the CamelCase choice configurable in case of people wanting to control that.